### ISSUE TITLE
Feature request: Astronomy schema

### ISSUE THREAD

**Issue by PolGuixe** [mantrajs/mantra-cli#67]
Would it be possible to add [Astronomy](https://github.com/jagi/meteor-astronomy) as an schema option when generating a collection?

**Comment by sungwoncho** [comment]
Yes, the schema option is extendable. We can add Astronomy and users will be able to run: `mantra g collection books -s astronomy`.

But I've never used Astronomy. Would you like to do a PR and get the ball rolling?

The relevant parts are: [here](https://github.com/mantrajs/mantra-cli/blob/master/lib/generators/collection.js#L18-L26), [here](https://github.com/mantrajs/mantra-cli/blob/master/lib/generators/utils.js#L119-L124), and [here](https://github.com/mantrajs/mantra-cli/blob/master/templates/lib/collections/generic.tt#L2-L4).

**Comment by PolGuixe** [comment]
Cool. I'll have a look and I will let you know.

**Comment by mbabauer** [comment]
I was looking at what it would take to implement Astronomy as well, but I don't think it was so simple.  The problem is that Collections2 and SimpleSchema seem to work by adding a `transform()` to the Collection, whereas Astronomy seems to create a Class that references your Collection.

For instance, a typical Astronomy setup might look like:

``` javascript
import { Class } from 'meteor/jagi:astronomy';
import { Widgets } from './collections';

export const Widget = Class.create({
    name: 'Widget',
    collection: Widgets,
    fields: {
        /* list of fields/options */
    }
});
```

So, you need to know both the class name and the name of the collection.

**Comment by PolGuixe** [comment]
@mbabauer you are right. You need to define both the name of the class and the name of collection. But as you creation the collection and the class at the same time, you can follow this convention: 
- Collection name in plural e.g. Items
- Class name in singular e.g. Item

Then the code that needs to be inserted can look like: 

```
import { Mongo } from 'meteor/mongo';
import { Class } from 'meteor/jagi:astronomy';


const Items = new Mongo.Collection('items');

const Item = Class.create({
  name: 'Item',
  collection: Items,
  fields: {
    name: {
      type: String,
   ....
    },
  }
});

export default Item;

```

**Comment by PolGuixe** [comment]
@sungwoncho could you have a lot at [this commit ](https://github.com/Zetoff/mantra-cli/commit/4a6f3f776d2129dfc308293ff738778ed7b20ca0). I think that I am not missing anything. If it is ok for you I'll create a PR.

**Comment by sungwoncho** [comment]
@PolGuixe  Looks good. I commented on [this line](https://github.com/Zetoff/mantra-cli/commit/4a6f3f776d2129dfc308293ff738778ed7b20ca0#diff-a154dfc9a20b2e49e7bef49887c05d7dR123).

**Comment by PolGuixe** [comment]
@sungwoncho cool. I have made the changes in this [commit](https://github.com/Zetoff/mantra-cli/commit/dc33cd7c89d5c35e252849f86f75957ddc3f186b). Although I am not sure I have used the right way import inflect with ES6.

**Comment by sungwoncho** [comment]
@PolGuixe let's make a PR and we will test it.

**Comment by PolGuixe** [comment]
PR added.

**Comment by sungwoncho** [comment]
I will test by this weekend. @mbabauer feel free to test it if you have time

**Comment by sungwoncho** [comment]
@PolGuixe I published a release candidate. Can you test v0.3.13-rc.2?

**Comment by PolGuixe** [comment]
@sungwoncho where I can find v0.3.13-rc.2? I can just see till v0.3.13-rc.1. Is the astronomy branch?

**Comment by sungwoncho** [comment]
@PolGuixe It's released on npm. On master branch.

**Comment by PolGuixe** [comment]
@sungwoncho on npm I can only se v0.3.**12**-rc2. ~~Using this version and running `mantra g collection books -s astronomy` it doesn't work properly. It just creates a standard collection without any type of schema.~~
Edit: I didn't installed v0.3.12-rc2 properly. Once installed it works as expected and generates the collection and the basic schema.

**Comment by sungwoncho** [comment]
Released [0.3.13](https://github.com/mantrajs/mantra-cli/blob/master/RELEASE_NOTE.md#0313-june-12-2016). Good work.
